packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js

```diff
@@ -157,7 +157,7 @@ export default class AgoraIpOrderCtrl {
   loadPrivateCloudIpOffers(serviceName) {
     return this.IpAgoraOrder.getPrivateCloudIpOffers(serviceName).then(
       (ipOffers) => {
-        this.blockIpOffers = ipOffers.map((offer) => {
+        const bo = ipOffers.map((offer) => {
```

Comment:
i guess better naming
```suggestion
        const ipBlockOffers = ipOffers.map((offer) => {
```